network/wsPeer.go

```diff
@@ -222,13 +237,44 @@ func (wp *wsPeer) Unicast(ctx context.Context, msg []byte, tag protocol.Tag) err
 	return err
 }
 
+// Respond submits the request to the server, waits for a response
```

Comment:
I can't see where does it wait for response, seems like outdated comment

Reply:
Correcting it to:
// Respond sends the response of a request message